_posts/2017/2017-02-28-Using common development environmet.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "One environment to rule them all"
+date: "2017-02-22 07:00:00"
+image: blog-header/ring.jpg
+categories: tech-stories
+authors: ["Jordi Viladoms"]
+---
+
+One of the most popular problems that developers have to face is to work with a common development environment. At the same time, this environment has to be the same that the production one. ePages software is working into servers with centOS or debian and it makes mandatory for the developers to write the code under this environment. For sure make no sense to force all the developers to have one of those OS installed at the laptop taking into account that the most preferred OS are [macOS, ubuntu or windows](https://developer.epages.com/blog/2016/12/22/the-battle-of-the-operating-systems.html).
```

Comment:
At the same time, this environment has to be the same as the production environment.
ePages software is working into servers with [CentOS](https://www.centos.org/) or [Debian](https://www.debian.org/) which makes it mandatory for us developers to write the code in this environment.
For sure, it doesn't make sense to force all the developers to have one of those OS installed on the laptop taking into account that the most preferred OS are [macOS, ubuntu or windows](https://developer.epages.com/blog/2016/12/22/the-battle-of-the-operating-systems.html).